src/Tribe/Attendees.php

```diff
@@ -859,4 +859,37 @@ public function user_can_manage_attendees( $user_id = 0, $event_id = '' ) {
 
 		return $user_can;
 	}
+
+	/**
+	 * Create attendee from Ticket data and validated attendee data.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $ticket Ticket Object.
+	 * @param array                         $attendee_data Validated attendee data.
+	 *
+	 * @return WP_Post|false
+	 */
+	public function create_attendee( $ticket, $attendee_data ) {
+
+		$provider = $ticket->get_provider();
+
+		/** @var Tribe__Tickets__Attendee_Repository $orm */
+		$orm      = tribe_attendees( $provider->orm_provider );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 6 spaces

<sub>hash: eda5a07ecaebac8b71bce27235fd045e</sub>